jenkins_build.sh

```diff
@@ -0,0 +1,20 @@
+set -e
```

Comment:
i'm fine with changing this however you guys see fit, or just removing entirely. i assume it's just here to halt the jenkins build in case any part of this script fails (so nothing weird happens when calling gcloud commands or something)